app/js/lib/foxbox/services/base.js

```diff
@@ -65,40 +64,46 @@ export default class BaseService extends EventDispatcher {
   }
 
   /**
-   * Call a service setter with a value.
+   * Sends a value to a service channel.
    *
-   * @param {Object|string} selector A value matching the kind of a channel.
-   * @param {string|number|boolean} value
+   * @param {Object|string} selector Selector for the channel to use.
+   * @param {*} value Value to send to the channel.
    * @return {Promise}
    */
-  set(selector, value = '') {
-    const setter = this[p.getSendChannel](selector);
-    const setterType = this[p.getSetterValueType](setter.kind);
-
-    return this[p.api].put(
-      'channels/set',
-      [[{ id: setter.id }, { [setterType]: value }]]
-    );
+  send(selector, value = null) {
```

Comment:
This library abstracts the communication with the box, it doesn't have to match perfectly the taxonomy. I like `set` and `get` better as there are easy to understand and to remember. I don't think they're incompatible with the term `channels`.
Anyway, I leave the final word to you if you think that's better to rename them.